crates/rolldown_error/Cargo.toml

```diff
@@ -16,5 +16,5 @@ workspace = true
 [dependencies]
 ariadne    = { workspace = true }
 napi       = { workspace = true, optional = true }
-oxc        = { workspace = true }
+oxc        = { workspace = true, features = ["sourcemap"] }
```

Comment:
You should enable the feature on the workspace level

Reply:
Done